radanalyticsio/spark/cluster/base/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spark-operator
+  labels:
+    app: jupyterhub
+    component: spark-operator
+    app.kubernetes.io/name: spark-operator
+    app.kubernetes.io/version: v0.1.5-v1alpha1
+spec:
+  replicas: 1
+  selector:
+    matchLabels: 
+      component: spark-operator
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        component: spark-operator
+    spec:
+      serviceAccountName: spark-operator
+      containers:
+      - name: spark-operator
+        image: quay.io/opendatahub/spark-operator:1.2.0
+        env:
+        - name: WATCH_NAMESPACE
+          value: "{{ meta.namespace }}"
```

Comment:
This variable won't work kfdef deployment.
I think the value is provided by OLM based on the OperatorGroup in the namespace
```
valueFrom:
  fieldRef:
    fieldPath: metadata.annotations['olm.targetNamespaces']
```